around/views.py

```diff
@@ -1,8 +1,10 @@
 from flask import render_template
 
 from core import app
+from user.models import User
 
 
 @app.route('/around/')
 def around():
-    return render_template('around/map.html')
+    users = User.objects.all()
```

Comment:
You should only send users that permitted to do so here instead of filtering in the template:
`users = User.objects(allow_community=True)`